### ISSUE TITLE
ipy.highlighting_style can be a string

### ISSUE THREAD

**Issue by njsmith** [tommikaikkonen/prettyprinter#7]
* PrettyPrinter version: 0.8.0

According to https://ipython.readthedocs.io/en/stable/config/details.html#terminal-colors:
> `TerminalInteractiveShell.highlighting_style` determines prompt colours and syntax highlighting. It takes **the name (as a string)** or class (as a subclass of `pygments.style.Style`) of a Pygments style, or the special value 'legacy' to pick a style in accordance with `InteractiveShell.colors`.

(emphasis added). But if I set `highlighting_style = "default"` in my IPython startup file, and enable prettyprinter integration, then I get a traceback ending in:

```
~/.user-python3.5-64bit/local/lib/python3.5/site-packages/prettyprinter/color.py in colored_render_to_stream(stream, sdocs, style, newline, separator)
    213                     except KeyError:
    214                         pygments_token = _SYNTAX_TOKEN_TO_PYGMENTS_TOKEN[sdoc.value]
--> 215                         tokenattrs = style.style_for_token(pygments_token)
    216                         color = styleattrs_to_colorful(tokenattrs)
    217                         color_cache[sdoc.value] = color

AttributeError: 'str' object has no attribute 'style_for_token'
```

I think there's some missing `if instance(style, str): style = pygments.styles.get_style_by_name(style)`

**Comment by tommikaikkonen** [comment]
Thanks! Should be fixed in 0.8.1 that I pushed to PyPi.

https://github.com/tommikaikkonen/prettyprinter/commit/11a4e1a64ece2d20e0596b3f4d3dc088af41db3e

**Comment by njsmith** [comment]
Yep, works for me, thanks!
